data/json/items/tool/lighting.json

```
@@ -232,6 +232,178 @@
     },
     "flags": [ "LIGHT_300", "CHARGEDIM", "TRADER_AVOID", "BELT_CLIP", "WATER_BREAK", "HELMET_HEAD_ATTACHMENT", "HEAD_STRAP_MOUNT" ]
   },
+  {
+    "id": "compact_tac_flashlight",
+    "type": "TOOL",
+    "//": "Based on the TLR-1, this can go on a pistol or a long-arm",
+    "looks_like": "flashlight",
+    "name": { "str": "compact tactical flashlight (off)", "str_pl": "compact tactical flashlights (off)" },
+    "description": "A flashlight designed to endure rough conditions.  You could attach it to a firearm with a mount.  This one is shorter than a regular flashlight.",
+    "material": [ "plastic", "aluminum" ],
+    "symbol": ";",
+    "color": "black",
+    "weight": "119 g",
+    "volume": "117 ml",
+    "longest_side": "86 mm",
+    "price": 500,
+    "price_postapoc": 100,
+    "charges_per_use": 1,
+    "ammo": [ "battery" ],
+    "flags": [ "BELT_CLIP", "WATER_BREAK_ACTIVE", "HELMET_HEAD_ATTACHMENT", "HEAD_STRAP_MOUNT", "GUNLIGHT" ],
```

```suggestion
    "flags": [ "WATER_BREAK_ACTIVE", "HELMET_HEAD_ATTACHMENT", "GUNLIGHT" ],
```

I buy being able to stick it on a helmet rail, but not a headstrap unless it's got a rail in the recipe, which it doesn't.

Also there's nothing on these that allows them to be attached on a belt clip.